DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
```

Comment:
Oh, that makes me sad: they're just lists of login methods, where the individual login methods are rendered slightly differently (links or buttons). Again, I think this points to incorrect decomposition of the partials.